test/transport/test_serial.py

```diff
@@ -55,10 +55,10 @@ async def test_external_methods(self, inx):
         comm.sync_serial.read = mock.MagicMock(return_value="abcd")
         comm.sync_serial.write = mock.MagicMock(return_value=4)
         comm.sync_serial.fileno = mock.MagicMock(return_value=2)
-        comm.sync_serial.async_loop.add_writer = mock.MagicMock()
-        comm.sync_serial.async_loop.add_reader = mock.MagicMock()
-        comm.sync_serial.async_loop.remove_writer = mock.MagicMock()
-        comm.sync_serial.async_loop.remove_reader = mock.MagicMock()
+        comm.async_loop.add_writer = mock.MagicMock()
```

Comment:
you need to update your dev, you are several commits behind.

Reply:
what do you mean - the diff wouldn't show if there wasn't a diff against dev